src/routes/queue.routes.js

```diff
@@ -1,57 +1,108 @@
 const express = require('express')
 const queue_routes = express.Router()
-const Redis = require('ioredis')
-const redis = new Redis()
 
-const queue = (client) => {
-  return{
+const healthyQueue = require('../queue/patient-queue')
 
-    push: async (key,val) => await client.rpush(key,val),
-    pop: async (key) => await client.lpop(key),
-    range: async (key) => await client.lrange(key,0,-1)
+queue_routes.get('/queue', async(req, res) => {
+  res.status(200).send(healthyQueue)
+})
+
+queue_routes.post('/queue/enqueue', async (req, res) => {
+
+  let priority = req.body.queue
```

Comment:
We might want a check here that the patient and visit already exist, but that can come in a follow up PR.